### PR TITLE
RichTextLabel add function remove_line

### PR DESCRIPTION
**Pull request by Dumbeldor**

### CODE REVIEW

File: scene/gui/rich_text_label.cpp [hunk 1]
```diff
@@ -1216,6 +1216,28 @@ void RichTextLabel::_add_item(Item *p_item, bool p_enter, bool p_ensure_newline)
 
 }
 
+void RichTextLabel::_remove_item(Item* p_item, const int line) {
```
Comment (bojidar-bg) [inline scene/gui/rich_text_label.cpp]: Doesn't match prototype, thus failing build

Comment (Dumbeldor) [inline scene/gui/rich_text_label.cpp]: Sorry i forgot to change it.
It's fix

**Comment by akien-mga** [comment]
It's best if you don't force push "TEMP" commits but just fix your local branch, and push it when it's good locally :)

**Comment by Dumbeldor** [comment]
Yes you're right sorry, it was just a forget about me. :D

**Review by akien-mga — APPROVED** [review]

**Pull request merged**